app/components/Common/InventoryCard/index.js

```diff
@@ -1,32 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { View, Text, StyleSheet, Image, Platform } from 'react-native';
 import { Colors, Typography } from '_styles';
 import { single_tree_png, placeholder_image, map_img } from '../../../assets';
 import MCIcons from 'react-native-vector-icons/MaterialCommunityIcons';
 import i18next from 'i18next';
 import RNFS from 'react-native-fs';
 import { INCOMPLETE, INCOMPLETE_SAMPLE_TREE } from '../../../utils/inventoryConstants';
+import { getCdnUrls } from './../../../actions/user';
 
 const InventoryCard = ({ data, icon, activeBtn, onPressActiveBtn, hideImage }) => {
+  const [imageSource, setImageSource] = useState();
+  useEffect(() => {
+    getCdnUrls(i18next.language).then((cdnMedia) => {
+      // console.log(cdnMedia, 'cdnMedia');
+    });
+    if (data.imageURL) {
+      const imageURIPrefix = Platform.OS === 'android' ? 'file://' : '';
+      setImageSource({
+        uri: `${imageURIPrefix}${RNFS.DocumentDirectoryPath}/${data.imageURL}`,
+      });
+    } else if (data.cdnImageUrl) {
+      setImageSource({
+        uri: `https://bucketeer-894cef84-0684-47b5-a5e7-917b8655836a.s3.eu-west-1.amazonaws.com/development/media/cache/coordinate/thumb/${data.cdnImageUrl}`,
```

Comment:
You can merge the develop branch and use `CDN_URL` from the environment now instead of https://bucketeer-894cef84-0684-47b5-a5e7-917b8655836a.s3.eu-west-1.amazonaws.com/development like at https://github.com/Plant-for-the-Planet-org/treemapper/blob/77924630f039c356cda2197b2f31854f74fddb7f/app/components/MainScreen/index.js#L296-L298 with the import as https://github.com/Plant-for-the-Planet-org/treemapper/blob/77924630f039c356cda2197b2f31854f74fddb7f/app/components/MainScreen/index.js#L48-L50